servers/src/mining/stratumserver.rs

```diff
@@ -539,6 +544,12 @@ impl StratumServer {
 				LOGGER,
 				"(Server ID: {}) Solution Found for block {} - Yay!!!", self.id, params.height
 			);
+			worker_stats.num_accepted += 1;
```

Comment:
This will be a double count since we already count the share below

Reply:
It depends on your other comment, but we can add the same logging as below and keep the return thus the count here. Since we might want to keep the ok response anyway. Or we could have a submit response message with the information that the share was a valid solution.

Reply:
Actually found a simpler way. Update incoming.